quadriga/src/main/java/edu/asu/diging/quadriga/domain/elements/TermPart.java

```diff
@@ -0,0 +1,65 @@
+package edu.asu.diging.quadriga.domain.elements;
+
+public class TermPart extends Element {
+
+    Long graphId;
```

Comment:
Is that required? all elements of a creationevent will be in the same mongo document, right? so no need for a graph id?